x-pack/metricbeat/module/azure/client.go

```diff
@@ -296,6 +192,7 @@ func (client *Client) GetMetricValues(referenceTime time.Time, metrics []Metric,
 		client.MetricRegistry.Update(metric, MetricCollectionInfo{
 			timeGrain: timeGrain,
 			timestamp: referenceTime,
+			//timestamp: time.Now().UTC(),
```

Comment:
test leftover?

Reply:
Yep, this was a leftover from a discarded solution. Thanks!